### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by janishiestand**

### CODE REVIEW

**Comment by Noel-Voelin** [comment]
@janishiestand I have an idea for your project that could be a great exercise. So basically, Now you have a db where you can add movies and stuff. Which is cool. But not all too useful. Now, There is an Open API that allows anybody to request information about a movie. 


It's called OMDb and it's really cool. So what you could do as part of your project is:

Instead of just displaying the movies that you have in your db, you could also add some kind of a search box that will actually load the movie from this API and add it to your collection.

Here is an article that explains how the API works: https://medium.com/nerd-for-tech/the-open-movie-database-omdb-for-fetch-movie-data-bc5ff46bec8

Basically, you just get a token and then you can request some huge Db with tonns of movie info.
